integration/integration_test.go

```diff
@@ -319,7 +319,7 @@ func TestMountsnoop(t *testing.T) {
 	mountsnoopCmd := &command{
 		name:           "Start mountsnoop gadget",
 		cmd:            fmt.Sprintf("$KUBECTL_GADGET mountsnoop -n %s", ns),
-		expectedRegexp: `test-pod\s+test-pod\s+mount.*mount\("/mnt", "/mnt", .*\) = -ENOENT`,
+		expectedRegexp: `test-pod\s+test-pod\s+mount.*mount\("/mnt", "/mnt", .*\) = -2`,
```

Comment:
I run the integration test on an AKS cluster and I was already getting `-2` even before this PR 🤔

Reply:
Yeah, seems like @eiffel-fl confirmed it, isn't it?

Reply:
Yes I also got the `-2` on the output instead of `-ENOENT`.
I prefer the `-E*` output but I think we will need to stick with the integer value.